trimesh/voxel/base.py

```diff
@@ -141,8 +141,10 @@ def strip(self):
 
     @caching.cache_decorator
     def bounds(self):
-        points = self.points
-        bounds = np.array([points.min(axis=0), points.max(axis=0)])
+        indices = self.sparse_indices
+        bounds = np.array([indices.min(axis=0) - 0.5,
+                           indices.max(axis=0) + 0.5])
+        bounds = self._transform.transform_points(bounds)
```

Comment:
I like the sparse logic for getting bounds! I think however you need to transform all 8 corners of the AABB then take the min-max of that *after* transform as min-max may switch around. Suggestion:
```
from .. import bounds as bounds_module

def bounds(self):
    indices = self.sparse_indices
    # get all 8 corners of the AABB
    corners = bounds_module.corners([indices.min(axis=0) - 0.5, indices.max(axis=0) + 0.5])
    # transform these corners to a new frame
    corners = self._transform.transform_points(corners)
    # get the AABB of corners in-frame
    bounds = np.array([corners.min(axis=0), corners.max(axis=0)])
```

Reply:
Fixed!

Reply:
Awesome thanks!